jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,287 @@
+    "    formated_properties.append(\"Absolute Maximum: \"+str(round(abs_max,3))+\" A,Absolute Minimum: \"+str(round(abs_min,3))+\" A,Mean Value: \"+str(round(abs_mean,3))+\" A\") \n",
+    "\n",
+    "#display(channels[0])\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "grp_1": 23.8,
+      "grp_2": 22.3
+     }
+    },
+    "tags": [
+     "Parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# --- The Analysis Automation Procedure Package parameter cell correspnding to the ANP parameter definition. \n",
+    "#     (see: cell \"property inspector\" => area: Advanced Tools => cell metadata)\n",
+    "\n",
+    "grp_1 = 22.3\n",
+    "grp_2 = 23.8"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# --- The  Analysis Automation Procedure Package parameter parameter set is avaible from the cell below the parameter cell itself \n",
+    "\n",
+    "# put the parameter to the info box of the plot. \n",
+    "formated_properties[0] +=\",Parameter: \" + str(grp_1)\n",
+    "formated_properties[1] +=\",Parameter: \" + str(grp_2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# -- Get bulk data from DataLink\n",
+    "bulk_data = []\n",
+    "file_id = data_links[0]['fileId']\n",
+    "\n",
+    "for group in get_groups(file_info):\n",
+    "    channel_specifier = []\n",
+    "    chn_specs = []\n",
+    "    group_name = group.name \n",
+    "    channels = get_channels(group)\n",
+    "    for channel in channels:\n",
+    "        channel_name = channel.name \n",
+    "        channel_specifier = OneChannelSpecifier(file_id=file_id, group_name=group_name, channel_name=channel_name)\n",
+    "        #display(channel_specifier)\n",
+    "        chn_specs.append(channel_specifier)\n",
+    "\n",
+    "    xy_chns = ChannelSpecificationsXyChannels(y_channels=chn_specs)\n",
+    "    channel_specs = ChannelSpecifications(xy_channels=[xy_chns])\n",
+    "    query = QueryDataSpecifier(channel_specs)\n",
+    "    #display(query)\n",
+    "    data = await data_api.query_data(query)\n",
+    "    \n",
+    "    # get numeric y data\n",
+    "    y_channels = data.data[0].y\n",
+    "    #display(len(y_channels))\n",
+    "    values = list(map(lambda c: c.numeric_data, y_channels))\n",
+    "    #display(len(values[0]))\n",
+    "    #display(values)\n",
+    "    bulk_data.append(values)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# --- Helper fuction and const to plot the data ---\n",
```

Comment:
Maybe: Helper _function_ and constant for plotting data ?

Reply:
fixed